modules/ServerAPI/src/client/AttachFile.coffee

```diff
@@ -102,14 +102,14 @@ class BasicFileController extends AbstractFormController
 		$(@el).empty()
 		$(@el).html @template(@model.attributes)
 		fileValue = @model.escape('fileValue')
-		urlValue = @model.excape('urlValue')
+		urlValue = @model.escape('urlValue')
```

Comment:
@brianbolt I believe this is purely a typo so I've corrected it without testing

Reply:
Whoops, good catch. I agree with the change.